### ISSUE TITLE
Allow configuration of multiple plot folders

### ISSUE THREAD

**Issue by mrpsion** [mrpsion/burst-mining-system#1]
It has been requested to add a feature to allow the configuration and use of multiple plot folders.

**Comment by bsom** [comment]
Yes, Please!

**Comment by wtfbob** [comment]
Most definitely needed

**Comment by jambrose** [comment]
Yes please! I tried creating soft links to my plot folder in Linux, but the system doesn't seem to like that approach. I ended up just pointing it at the folder on my other hard drive. Not sure what to do if I add another hard drive to this.
